index.html

```diff
@@ -97,14 +122,14 @@ <h1>Multi-Post
                     <button>javascript</button>
                 </ul>
                 <button class="see_project">See project</button>
-
+            </div>
             </div>
         </section>
     </main>
     <!-- footer -->
     <footer>
-        <section class="section1">
-            <h1>About Myself</h1>
+        <div class="section1">
+            <h3>About Myself</h3>
```

Comment:
- [ ] I would recommend you to work more on your about section in order to match the figma design. As stated by the previous reviewer, the about part takes up 40% width while the language part takes up the remaining 60%. As such the margin left and right must be the same. Kindly increase the width for the language part and make the margin at the both sides to be equal. Take a look :point_down: 

### Your Design
![your-about](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/21ca2e29-4655-4b29-92ec-6d672456285b)

### Figma Design
![figma-about](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/7f914959-432c-4867-80d5-70a2d6c54d25)

- [OPTIONAL] Kindly look into the way your `Get my resume` button appears and make it look like how it it on the figma template.

### Your Design
![your-getmyresume](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/0406e026-0533-4aa2-a013-707035792864)

### Figma Design 
![fig-getmyresume](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/29ad22c1-3ebb-4e37-af5e-4a3abff38cf0)